src/Autosuggest.js

```diff
@@ -215,6 +249,8 @@ export default class Autosuggest extends Component {
       focusedSectionIndex: sectionIndex,
       focusedSuggestionIndex: suggestionIndex
     });
+
+    this.onSuggestionFocused(sectionIndex, suggestionIndex);
```

Comment:
I don't think so. When the mouse enters we should call `onSuggestionFocused`, and when the mouse leaves we should call `onSuggestionUnfocused`. That is why the tests are working. We could implement all of the logic inside this handler though.